docs/online/src/scenarios/postmessage.rst

```diff
@@ -397,6 +465,35 @@ every outgoing PostMessage:
             }
         }
 
+..  data:: App_RefreshSessionInfo
+
+    The :term:`App_RefreshSessionInfo` message is poseted when the app detects the need of new tokens. The host responds to this by
+    sending a response message :term:`App_RefreshSessionInfo` with a :term:`Host_RefreshSessionInfo` response. 
+
+    ..  note::
+        If the host does not return the :term:`SupportsSessionRefresh` parameter in their :ref:`CheckFileInfo` response, then
+        this will not be available in |wac|.
```

Comment:
WAC will understand that the host doesn't support token refresh and will not request new tokens